src/Sonarr.Api.V3/ProviderControllerBase.cs

```diff
@@ -45,11 +45,11 @@ protected override TProviderResource GetResourceById(int id)
         [Produces("application/json")]
         public List<TProviderResource> GetAll()
         {
-            var providerDefinitions = _providerFactory.All().OrderBy(p => p.ImplementationName);
+            var providerDefinitions = _providerFactory.All();
 
-            var result = new List<TProviderResource>(providerDefinitions.Count());
```

Comment:
Here we actually ordered definitions twice - the first time during Count() and the second one in the loop.